hack/verify-upstream-commits.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd $(dirname "${BASH_SOURCE}")/..
+
+ok_revert_pat='UPSTREAM: [0-9]{4,}: revert origin [a-f0-9]{7,}:'
```

Comment:
Here are some real examples since the last rebase:

```
8156eea UPSTREAM: revert: 199adb7: <drop>: add back flag types to reduce noise during this rebase
6105be9 UPSTREAM: revert: 0048df4: <carry>: Disable --validate by default
98fdf96 UPSTREAM: 16032: revert origin 03e50db: check if /sbin/mount.nfs is present
```

Here's what you and @pmorie agreed upon in an old list thread from July:

```
UPSTREAM: <upstream PR number>: revert origin <commit being reverted>: <message of commit being reverted>
```

Sounds like we need to come to some consensus on the revert format...